modules/media/ajax/FileUpload.php

```diff
@@ -414,9 +414,9 @@ function checkDateTaken($dateTaken)
         }
 
         $now  = new DateTime();
-        $diff = date_diff($date, $now)->format("%a");
-        if ($diff > 0) {
-            showMediaError("Date of administration can not be in the future", 400);
+        $diff = date_diff($date, $now)->format("%R%a");
```

Comment:
@jesscall the uppercase `R` will return either `+5` or `-5` both being strings and should not be compared with an int as done below in the if statement

use `intval` to convert to integer

https://www.php.net/manual/en/function.intval.php